build.sbt

```diff
@@ -50,6 +55,29 @@ lazy val `arrows-stdlib` =
 lazy val `arrows-stdlib-jvm` = `arrows-stdlib`.jvm
 lazy val `arrows-stdlib-js` = `arrows-stdlib`.js.settings(test := {})
 
+lazy val `arrows-stdlib-cats` =
```

Comment:
What is the convention for these sub-projects? What is `stdlib` referring to? Couldn't it be just `arrows-cats`?

Reply:
`stdlib` refers to the use of stdlib `Future` wheras `arrows-twitter` uses twitter `Future`.